fluidsynth/src/sfloader/fluid_defsfont.h

```diff
@@ -435,7 +435,7 @@ struct _fluid_defpreset_t
 {
   fluid_defpreset_t* next;
   fluid_defsfont_t* sfont;                  /* the soundfont this preset belongs to */
-  char name[21];                        /* the name of the preset */
+  char name[22];                        /* the name of the preset */
```

Comment:
> Really 2 bytes (unsigned short).

Yep, sry, noticed too late.

> we shouldn't use longer names than the SF2 spec permits (at least not only for the purpose of fitting this default string).

`SFPreset` is the struct used for reading soundfonts. This correctly uses [char name[21]](https://github.com/FluidSynth/fluidsynth/blob/master/fluidsynth/src/sfloader/fluid_defsfont.h#L129). What we make up internally with `_fluid_defpreset_t` shouldnt matter that much, should it? One would always use `strlen() and its friends` and not just silently assume there are max. 21 chars IMO.